pulsar-client/src/main/java/org/apache/pulsar/client/impl/TransactionMetaStoreHandler.java

```diff
@@ -167,6 +167,8 @@ public CompletableFuture<Void> connectionOpened(ClientCnx cnx) {
                     return null;
                 });
             } else {
+                LOG.warn("Can not Connect to the transaction coordinator because the protocol version {} is "
```

Comment:
I saw the compare line is `if (cnx.getRemoteEndpointProtocolVersion() > ProtocolVersion.v18.getValue()) `, why log "lower than 19?"